Unit test fails in golang:1.13-alpine due to path assumption

# 🐞 Bug Report

### Affected Services
security-file-token-provider (security-secretstore-setup)

### Description and Minimal Reproduction

 Currently in the Jenkins job, the tests run on the physical host. In the new pipeline everything runs in the context of the `golang:1.13-alpine` image. When I switched to this the go test command started failing in this test:

```
00:03:26.562  --- FAIL: TestCreatesFile (0.00s)
00:03:26.562      tokenprovider_linux_test.go:55: 
00:03:26.562          	Error Trace:	tokenprovider_linux_test.go:55
00:03:26.562          	Error:      	Received unexpected error:
00:03:26.562          	            	open /tmp/tokenprovider_linux_test.dat: no such file or directory
00:03:26.562          	Test:       	TestCreatesFile
00:03:26.562  FAIL
00:03:26.562  coverage: 26.2% of statements
00:03:26.562  FAIL	github.com/edgexfoundry/edgex-go/internal/security/secretstore	0.034s
```

After lots of trial and error I traced to the issue to this line in this file: internal/security/secretstore/tokenprovider_linux_test.go
```
TokenProvider:     "/usr/bin/touch"
```

```
docker run --rm -it golang:1.13-alpine sh
$ /usr/bin/touch
sh: /usr/bin/touch: not found
```
In alpine touch is not at this path.

--- a/internal/security/secretstore/tokenprovider_linux_test.go
+++ b/internal/security/secretstore/tokenprovider_linux_test.go
@@ -34,7 +34,7 @@ import (
 func TestCreatesFile(t *testing.T) {
 	const testfile = "/tmp/tokenprovider_linux_test.dat"
 	config := secretstoreclient.SecretServiceInfo{
-		TokenProvider:     "/usr/bin/touch",
+		TokenProvider:     "touch",
 		TokenProviderType: OneShotProvider,
 		TokenProviderArgs: []string{testfile},
 	}

